google-api-client-bom/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.api-client</groupId>
+  <artifactId>google-api-client-bom</artifactId>
+  <version>1.26.1-SNAPSHOT</version><!-- {x-version-update:google-api-client-bom:current} -->
+  <packaging>pom</packaging>
+
+  <name>Google API Client Library for Java BOM</name>
+  <url>https://github.com/googleapis/google-api-java-client/tree/master/google-api-client-bom</url>
+  <description>
+    BOM for Google API Client Library for Java
+  </description>
+
+  <organization>
+    <name>Google LLC</name>
+  </organization>
+
+  <scm>
+    <connection>scm:git:https://github.com/googleapis/google-api-java-client.git</connection>
+    <developerConnection>scm:git:git@github.com:googleapis/google-api-java-client.git</developerConnection>
+    <url>https://github.com/googleapis/google-api-java-client</url>
+  </scm>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>sonatype-nexus-snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>sonatype-nexus-staging</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <site>
+      <id>github-pages-site</id>
+      <name>Deployment through GitHub's site deployment plugin</name>
+      <url>site/google-cloud-bom</url>
+    </site>
+  </distributionManagement>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.api-client</groupId>
+        <artifactId>google-api-client</artifactId>
+        <version>1.26.1-SNAPSHOT</version><!-- {x-version-update:google-api-client:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.api-client</groupId>
+        <artifactId>google-api-client-android</artifactId>
+        <version>1.26.1-SNAPSHOT</version><!-- {x-version-update:google-api-client:current} -->
```

Comment:
(I only realized this after looking at the corresponding review for oauth) the artifactId is wrong in the x-version-update strings.

Reply:
In this repo, we're only tracking google-api-client, google-http-client, and google-oauth-client. We use the same x-version-update for simplicity as each artifact because they are all versioned the same.